tide_media.info.yml

```diff
@@ -142,6 +142,7 @@ config_devel:
     - field.storage.media.field_media_topic
     - field.storage.media.field_media_transcript
     - field.storage.media.field_media_video_embed_field
+    - field.storage.media.field_media_site
```

Comment:
This field should be defined in the optional property because Tide Site is not a dependency of Tide Media.

Reply:
I reverted this change. it causes a bunch of dependency issues. 😫